stock_manual_shipping_weight/README.rst

```diff
@@ -0,0 +1,84 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+	:target: http://www.gnu.org/licenses/agpl
+	:alt: License: AGPL-3
+
+
+============================
+Stock manual shipping weight
+============================
+This module allows you to edit the weight for shipping field through a wizard.
+
+
+Installation
+============
+To install this module, you need to:
+
+Just install.
+
+
+Configuration
+=============
+
+To configure this module, you need to:
+
+#. Just install.
+
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. Choose any inventory transfer.
+#. Once in the transfer view, go to the "Additional Info" tab
```

Comment:
I would specify that you're talking about the form view.

Reply:
Done